### DISCUSSION TITLE
Unable to login due to MFA

### DISCUSSION THREAD

**Discussion by atulmaurya170** [community/community#23559] [New to GitHub]
<p>Hello,</p>
<p>I have an old account with GitHub (atulmaurya27 (email- <a href="mailto:atulmaurya27@gmail.com">atulmaurya27@gmail.com</a>)). I remember the password of the account as well, however the associated email is deactivated. I had a recent change in my laptop and I am unable to access my old laptop.</p>
<p>When I try to log in using my user account an OTP is send to my old email address which is not active. And due to this I am unable to access my account.</p>
<p>Please suggest some work-around as there is some critical code in my old account and I need to recover it.</p>
<p>Thanks,<br>
Atul</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>You’ll have to contact <a href="https://support.github.com/request" class="inline-onebox">GitHub Support</a> about that, account problems are not something the community can help with. Good luck!</p>
